src/main/webapp/script.js

```diff
@@ -1,28 +1,139 @@
-// Copyright 2020 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Do you like guacamole?', 'I went to six different schools', 'Are you also a cat person?', '你好呀朋友'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+
+//habit list item array
```

Comment:
Nit: I think you named your variable well enough that this comment isn't needed. Revaluate your other comments as well.

Also, typically comments have proper grammar and have a space after the `//`.

So as an example
`//create a new habit item based on the text from input`
`// Create a new habit item based on the text from input.`

Reply:
Thank you so much for all the advice for the comments and logs! I really needed the guidance.